src/app.js

```diff
@@ -0,0 +1,84 @@
+// Debemos poder leer un archivo.
+// Asegurarnos que sea un archivo Markdown.
+// Obtener los links de este archivo
+// Validar esos links, que me devuelva un status 200
+
+const {validateLinks, linkStats, validateStats } = require('./main.js');
+
+const fs = require('fs');
+const marked = require('marked');
+const path = require('path');
+//const colors = require('colors');
+
+let index = process.argv.indexOf("--file");
+let flags = process.argv;
+
+function readFile(uri) { 
+    if (fs.existsSync(uri)){
+        let fileContent = fs.readFileSync(uri, 'utf8');
+        return fileContent;
+    } else {
+        console.log('El archivo'.red, uri.red.bgCyan, 'NO existe'.red);
+        return false;
+    };
+};
+
+function getUri() {
+    let uri = process.argv[index + 1];
+    let ext = path.extname(uri);
+        if (index < 0) {
+             console.log('Poner'.red, '--file'.red.bgCyan, '(--file Nombre_Archivo.md)'.red);
+             return false;
+        } else if (ext != '.md') {
+             console.log('Archivo invalido!'.red + '\nDebe ser un archivo '.red + 'Markdown (.md)'.red.bgCyan);
+
+             return false;
+        } else {
+            return uri;
+        }
+}
```

Comment:
tener una mejor definición del propósito de la función 
```suggestion
function confirmFile() {
    let uri = process.argv[index + 1];
    let ext = path.extname(uri);
        if (index < 0) {
             console.log('Poner'.red, '--file'.red.bgCyan, '(--file Nombre_Archivo.md)'.red);
             return false;
        } else if (ext != '.md') {
             console.log('Archivo invalido!'.red + '\nDebe ser un archivo '.red + 'Markdown (.md)'.red.bgCyan);

             return false;
        } else {
            return uri;
        }
}
```